modular_skyrat/modules/berets/code/modules/clothing/head/jobs.dm

```diff
@@ -144,3 +144,11 @@
 	name = "head of personnel's beret"
 	desc = "A fancy beret designed by NT's Personnel division for their favorite head's head. This one is made out of white fabric. Fancy"
 	greyscale_colors = "#FFFFFF#D60000"
+
+/obj/item/clothing/head/beret/color
+	name = "beret"
+	desc = "A beret, perfect for war veterans and dark, brooding, anti-hero mimes."
```

Comment:
and this has the black beret description when it doesnt make sense when its not black

Reply:
I did scratch the "black" out of the description. One can be a multicolour war veteran.

The inheritance stuff you pointed out is exactly the same for the black beret (.../beret/black), so I left that in there before something I don't know about breaks.